### ISSUE TITLE
VoiceOver reads two words together for first element in quick open

### ISSUE THREAD

**Issue by thien-do** [microsoft/vscode#133649]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.58.2 (Universal)
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:09:06.581Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 21.1.0

Steps to Reproduce:

1. Press Command + P to open "quick open" panel
2. Type on some path
3. Let the screen reader (in this case voice over) reads the content:

![image](https://user-images.githubusercontent.com/5953369/134447764-fc3457ed-b736-4e93-bab2-f2e06b5e156c.png)

the image shows that there is no space between the path and the file name, so the screen reader cannot separate those two and pronounce character by character, instead of word by word like it should be

**Comment by isidorn** [comment]
Chrome upstream https://bugs.chromium.org/p/chromium/issues/detail?id=1289592
I can repro with Electron 16.

**Comment by isidorn** [comment]
Can you still reproduce this with the latest version of VS Code?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
